app/dash_apps/reaction_viewer/index.py

```diff
@@ -286,6 +297,49 @@ def display_value(value):
             except:
                 pass
 
+            try:
+                citation = ds.data.metadata["citations"][0]["acs_citation"]
+                citation = citation.replace("</em>", "*").replace(
+                    "<em>", "*"
+                )  # Italics 1
+                citation = citation.replace("</i>", "*").replace(
+                    "<i>", "*"
+                )  # Italics 2
+                citation = citation.replace("</b>", "**").replace("<b>", "**")  # Bold
+                citation = citation.replace(", ***", "*, **").replace(
+                    "***, ", "**, *"
+                )  # HTML vs Markdown corrections
+                doi = ds.data.metadata["citations"][0]["doi"]
+
+                citation = dedent(
+                    f"""
+                        ### Citation:
+                        {citation}
+
+                        **DOI**: [{doi}](https://doi.org/{doi})
+                        """
+                )
+            except:
+                citation = dedent(
+                    f"""
+                    ## Citation:
+                    *No information available.*
+                    """
+                )
+
+            mqcas_help = dedent(
+                """
+                We are still filling in data and expanding these Webapps.
+                If you have features suggestions or bug reports please file them [here](https://github.com/MolSSI/QCArchiveWebapps/issues/new/choose)
+                or if you wish to expand the datasets, methods, or bases please open an issue
+                [here](https://github.com/MolSSI/MQCAS/issues/new/choose).
+                """
+            )
```

Comment:
Consider putting this in its own card so it doesn't have to update every time we change the dataset.

Reply:
Yea, its easier this way for now (IMO). Its pretty lightweight compared to anything else.